docs_user/modules/proc_migrating-mon-and-mgr-from-controller-nodes.adoc

```diff
@@ -252,8 +219,9 @@ mon_host = [v2:172.16.11.54:3300/0,v1:172.16.11.54:6789/0] [v2:172.16.11.121:330
 12: vlan14    inet 172.16.14.223/24 brd 172.16.14.255 scope global vlan14\       valid_lft forever preferred_lft forever
 ----
 
-On the oc0-ceph-0:
-
+. On the oc0-ceph-0: 
```

Comment:
on the `oc0-ceph-0`, add the ip address of the mon that has been deleted from controller-0.

`$ sudo ip a add 172.16.11.121 dev vlan11`

and verify that the ip address has been assigned and can be reached:

`$ ip -o -4 a`